book.toml

```diff
@@ -3,7 +3,7 @@ title = "Asynchronous Programming in Rust"
 authors = ["Taylor Cramer"]
 
 [build]
-create-missing = false
+create-missing = true
```

Comment:
Why `create-missing = true` is needed?

Reply:
I think this is a mistake when I pull from upstream.  I will revert it.